.travis.yml

```diff
@@ -14,6 +14,7 @@ env:
     - LARAVEL_VERSION=5.7.*
     - LARAVEL_VERSION=5.8.*
     - LARAVEL_VERSION=6.0.*
+    - LARAVEL_VERSION=6.1.*
```

Comment:
Do we care about testing all the minor versions of 6? Now that they're following semver, we should be fine with just `6.*`